hyp3_gamma/water_mask.py

```diff
@@ -54,9 +54,10 @@ def get_water_mask_gdf(extent: geometry.Polygon) -> gpd.GeoDataFrame:
         GeoDataFrame of the water mask for the given extent
     """
     mask_location = 'asf-dem-west/WATER_MASK/GSHHG/hyp3_water_mask_20220912'
-    corrected_extent = split_geometry_on_antimeridian(json.loads(shapely.to_geojson(extent)))
+    # extent_poly = geometry.shape(extent)
+    corrected_extent = split_geometry_on_antimeridian(extent)
 
-    filters = list(set([('lat_lon', '=', get_geo_partition(coord)) for coord in extent.exterior.coords]))
+    filters = list(set([('lat_lon', '=', get_geo_partition(coord)) for coord in geometry.shape(extent).exterior.coords]))
```

Comment:
This line is 1 character too long. Also, Shapely is imported at the top of the file, but it is unused and should be removed.
```suggestion
    filters = list(set([('lat_lon', '=', get_geo_partition(coord)) for coord in geometry.shape(extent).exterior.coords]))
```